js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
```

Comment:
The `fieldId` property needs to match the `id` attribute of the the form controls, in this case the `Checkbox`. There is also already a control with the id `pattern`, so this should not be re-used.

Reply:
renamed "targetContextAttributes"
Notes: I've tested it locally, and in fact the Checkbox Id does not matter, but the Controller name has to match with the new name "targetContextAttributes"

Reply:
The checkbox `id` needs to match, as otherwise the input is not associated with the label. This creates problems for visually impaired users, as their accessibility tools are not able to understand the structure. It also handles other behavior, such as focusing the input when the label is clicked.